plugin/web/extensions/geolocation/core/index.tsx

```diff
@@ -0,0 +1,30 @@
+import { Icon } from "@web/sharedComponents";
+import { styled } from "@web/theme";
+
+import useHooks from "./hooks";
+
+const GeolocationWrapper: React.FC = () => {
+  const { handleFlyToCurrentLocation } = useHooks();
+
+  return (
+    <Wrapper width={44} height={44}>
+      <Icon icon="sun" size={20} onClick={handleFlyToCurrentLocation} />
+    </Wrapper>
+  );
+};
+
+export default GeolocationWrapper;
+
+const Wrapper = styled.div<{ width?: number; height?: number }>`
```

Comment:
Since width and height don't change you don't need to pass the width/height as props.

Reply:
done

Reply:
fc8de4437577d111f90b0a1c7db09d37c48d4de6